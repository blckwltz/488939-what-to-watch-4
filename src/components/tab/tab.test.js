import React from 'react';
import renderer from 'react-test-renderer';
import Tab from './tab';

const mocks = {
  activeTab: `Overview`,
  tabLabel: `Details`,
  onClick: () => {},
};

it(`Should render Tab component correctly`, () => {
  const {activeTab, tabLabel, onClick} = mocks;
  const tree = renderer
    .create(
        <Tab
          activeTab={activeTab}
          tabLabel={tabLabel}
          onClick={onClick}
        />
    ).toJSON();

  expect(tree).toMatchSnapshot();
});
