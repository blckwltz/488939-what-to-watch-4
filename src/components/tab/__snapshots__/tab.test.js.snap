// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render Tab component correctly 1`] = `
<li
  className="movie-nav__item false"
  onClick={[Function]}
>
  <a
    className="movie-nav__link"
    href="#"
  >
    Details
  </a>
</li>
`;
